.goreleaser.yml

```diff
@@ -72,6 +72,22 @@ dockers:
     use: "buildx"
     build_flag_templates:
       - "--platform=linux/arm64"
+  - image_templates:
+      - &amd_image_gh "ghcr.io/authzed/spicedb:v{{ .Version }}-amd64"
+    dockerfile: *dockerfile
+    goos: "linux"
+    goarch: "amd64"
+    use: "buildx"
+    build_flag_templates:
+      - "--platform=linux/amd64"
+  - image_templates:
+      - &arm_image_gh "ghcr.io/authzed/spicedb:v{{ .Version }}-arm64"
+    dockerfile: *dockerfile
+    goos: "linux"
+    goarch: "arm64"
+    use: "buildx"
+    build_flag_templates:
+      - "--platform=linux/arm64"
```

Comment:
You should be able to remove the config blocks for the gh images and just add a new tag to the existing image configs, like:

```yaml
- image_templates:
      - &arm_image "quay.io/authzed/spicedb:v{{ .Version }}-arm64"
      - &arm_image_gh "ghcr.io/authzed/spicedb:v{{ .Version }}-arm64"
```

Reply:
Ah you're right https://goreleaser.com/customization/docker/#publishing-to-multiple-docker-registries

Didn't see the gh images in the goreleaser output so assumed it needed to be built separately.